### PR TITLE
Grammar and Spelling fixes.

### PR DESCRIPTION
**Pull request by ghost**
@JMW2020

Grammar and Spelling fixes

5:15 PM 11/19/2020

### CODE REVIEW

File: nboard [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env python3
 
-
+# @JMW2020 5:15 PM CST 11/19/2020 Edit "Grammar and spelling fixes."
```
Comment (xfnw) [inline nboard:3]: no need to stick attribution into the file, its already on the git history and on the contributors tab in github

File: nboard [hunk 2]
```diff
@@ -122,25 +122,25 @@ if __name__ == "__main__":
...
 
-hold ctrl while pressing the arrow keys to go faster
+Hold ctrl while pressing the arrow keys to go faster
 
-bugs and stuff:
-please report any bugs to ~xfnw on IRC, anywhere you can find him, or
-on the tildegit page, https://tildegit.org/xfnw/nboard
+Bugs And Stuff:
+Please report any bugs to ~xfnw on IRC, anywhere you can find him, or
+On the tildegit page, https://tildegit.org/xfnw/nboard
```
Comment (xfnw) [inline nboard:143]: `On` has no reason to be capitalized here, the sentence continues from the previous line

**Review by xfnw — CHANGES_REQUESTED** [review]
nice overall, but some things need to be fixed

File: nboard (hunk 1 above)
Comment (ghost) [reply nboard:3]: @xfnw Ok!

File: nboard (hunk 2 above)
Comment (ghost) [reply nboard:143]: @xfnw Got it!

**Comment by xfnw** [comment]
closing since submitter deleted their account(?)

**Pull request closed without merging**